qutebrowser/utils/version.py

```diff
@@ -388,6 +395,11 @@ def version():
     for name, path in sorted(_path_info().items()):
         lines += ['{}: {}'.format(name, path)]
 
+    lines += [
+        '',
+        'Uptime: {}'.format(str(_uptime()))
```

Comment:
I added a trailing comma here and did remove the `str()` (as calling `.format(obj)` implicitly does `str(obj)` unless the object overrides how it wants to be formatted).